### PR TITLE
eth: made changes to conform better to the golint standards

### PR DESCRIPTION
**Pull request by williambannas**
minor syntactical changes

### CODE REVIEW

File: eth/config.go [hunk 1]
```diff
@@ -71,8 +71,7 @@ func init() {
 	}
 }
 
-//go:generate gencodec -type Config -field-override configMarshaling -formats toml -out gen_config.go
-
+//Config is go:generate gencodec -type Config -field-override configMarshaling -formats toml -out gen_config.go
```
Comment (karalabe) [inline eth/config.go]: Hah no, this is a special statement that is interpreted by `go generate` to generate source files. Please don't modify this.

File: eth/tracers/internal/tracers/assets.go [hunk 2]
```diff
@@ -72,17 +72,17 @@ func (fi bindataFileInfo) Sys() interface{} {
...
...da\x7f\x93\x54\xfa\xaa\xd2\x7f\x5e\x15\xe3\xd8\x75\xee\xb0\x2b\xed\x26\x0f\x5f\x19\x66\x6b\x65\xd8\xc3\x3e\x02\x4a\x2d\xef\x6f\x87\xf0\x9f\xc2\x13\x24\xc2\xc8\x67\xa2\x15\xbe\xf6\x77\x0e\x2b\x63\xd5\x26\x3b\x57\xca\xa1\xf7\x81\x51\x50\x42\xcc\x6d\xd6\xee\x77\x3a\x9d\x9f\xf1\xf8\x2c\xc2\xba\x7f\x15\x6b\xbd\xb7\xaa\x90\xb5\x59\x7c\x87\x0f\xf0\x06\x54\x12\x17\xaa\x13\x87\xf6\xbc\x4d\xda\xcf\x41\x87\xeb\x1f\x3f\xc0\xa8\x72\x59\x42\xdc\x26\xc9\x8f\x30\xde\xd8\x97\xca\x08\x22\x0b\x41\xb0\xce\xdd\x21\xf6\xbc\xa9\xda\x01\x24\xaa\xb0\xde\xc3\xa8\x13\x05\x6a\xdd\x51\xa7\x8a\xa7\x56\x4b\x22\x8a\x8c\x8e\xe5\xb2\x4b\xab\x4f\x02\x21\xa9\x10\x59\xa5\x10\xfe\xbc\xb1\x09\x08\x53\x8b\x28\x29\x97\x75\x23\xd8\xff\x50\x36\x50\xbb\x70\xe8\x7f\xe4\x83\x93\xc7\x7e\x6a\x3d\x85\x35\xbf\x42\xee\x29\x42\x27\xf8\x3b\xc7\x6e\xab\xae\xaa\xe6\x64\x80\x2b\xc7\x1f\xe7\xbf\x02\xae\x76\x15\x2f\x8c\xb0\x47\x1b\xe5\xf9\x11\x29\x49\xfb\x17\x1d\xd7\xfd\x6b\x0b\x1e\x99\x5c\x43\xee\x59\x10\x99\xb7\x55\x55\x24\xed\x63\x6d\xf2\x82\xe2\x0c\xcd\x9a\x52\xf8\xf8\x5c\xa0\xa3\x9c\x97\x89\x7e\xbe\x1b\xc1\x49\x14\xf2\xfc\xd6\xba\x3b\xea\xbc\x9e\x2b\x75\x07\x97\x3d\xfb\xd4\xfc\x6f\x00\x00\x00\xff\xff\x08\x1e\xd9\xac\x67\x0b\x00\x00")
```
Comment (karalabe) [inline eth/tracers/internal/tracers/assets.go]: What did you generate these files with? This is auto generated code, you can't modify it.

File: eth/config.go (hunk 1 above)
Comment (williambannas) [reply eth/config.go]: thanks ill change it back

File: eth/tracers/internal/tracers/assets.go (hunk 2 above)
Comment (williambannas) [reply eth/tracers/internal/tracers/assets.go]: I read that comment at the beginning and just thought it applied to the comment up top, I didn't generate this file at all, only took out the underscores, and made it camelcase, I will also revert all the changes from this file as well

Comment (williambannas) [reply eth/tracers/internal/tracers/assets.go]: thank you for your help

**Pull request merged**